### PR TITLE
removed DELTA_P variable from madx

### PR DESCRIPTION
**Pull request by JoschD**

### CODE REVIEW

File: omc3/correction/response_madx.py [hunk 1]
```diff
@@ -111,18 +112,19 @@ def _generate_madx_jobs(
...
+            current_job += f"twiss, file='{str(temp_dir / 'twiss.0')}'{delta_p_command};\n"
             
             if compute_deltap: # If DELTAP_NAME is in variables, we run this in the last iteration
                 # Due to the match and correction of the orbit, this needs to be run at the end of the process
                 incr_dict[DELTAP_NAME] = delta_k
-                current_job += f"{DELTAP_NAME} = {delta_k:.15e};\n" # Set deltap to delta_k
-                current_job += accel_inst.get_update_deltap_script() # This will add accel_inst.dpp to deltap and do twiss, correct, match
+                accel_inst.dpp += delta_k
+                current_job += accel_inst.get_update_deltap_script() # This will do twiss, correct, match at the updated dpp
                 current_job += f"twiss, deltap={DELTAP_NAME}, file='{str(temp_dir/f'twiss.{DELTAP_NAME}')}';\n"
```
Comment (jgray-19) [inline omc3/correction/response_madx.py:126]:
```suggestion
                current_job += f"twiss, deltap={accel_inst.dpp:.15e}, file='{str(temp_dir/f'twiss.{DELTAP_NAME}')}';\n"
```

File: omc3/model/accelerators/lhc.py [hunk 2]
```diff
@@ -502,13 +502,12 @@ def get_update_correction_script(self, outpath: Path | str, corr_files: Sequence
     
     def get_update_deltap_script(self) -> str:
```
Comment (jgray-19) [inline omc3/model/accelerators/lhc.py:503]: Could have a check for 0 self.dpp here, to skip everything

**Review by jgray-19 — COMMENTED** [review]
I started this, so I thought I would finish

**Comment by JoschD** [comment]
made better suggestions in another branch which has now been merged. Closing

**Pull request closed without merging**